lib/src/model/booking/booking.dart

```diff
@@ -86,7 +78,7 @@ abstract class Booking extends DateRangeItem {
 
   @override
   String toString() =>
-      [if (isLocked) '🔒', description, dateTimeRange].join(' ');
+      [if (isLocked) '🔒', description, textualDateTime].join(' ');
```

Comment:
This method should have been called here instead of just referencing its name. Addressed in #170.